test/ng/directive/ngClassSpec.js

```diff
@@ -462,6 +462,47 @@ fdescribe('ngClass', function() {
     expect(e2.hasClass('odd')).toBeFalsy();
   }));
 
+
+  it('should keep track of old classes even if odd/even does not match `$index` atm',
```

Comment:
This description makes hard to understand what is actually being tested.

Are you saying that ngClass is not removing old classes when both the $index and the expression value changes?